test/src/808-lease_failure/main

```diff
@@ -0,0 +1,74 @@
+#
+# Test for the cvmfs_server transaction command returns codes
+# 
+# This sets up a repository with gateway and simulates the case
+# when another release manager has already taken the lock for 
+# the path we want.
+
+cvmfs_test_name="Repository gateway"
+cvmfs_test_autofs_on_startup=false
+cvmfs_test_suites="quick"
+
+clean_up() {
+    echo "Cleaning up"
+
+    echo "  Removing output directories"
+    rm -rvf /tmp/cvmfs_out_*
+}
+
+
+run_transactions() {
+    set_up_repository_gateway
+
+    ## Transaction 1 (Creating a new deep directory sub-tree in place of a former file)
+    echo "  Starting transaction 1"
+    cvmfs_server transaction test.repo.org
+    if [ $? -ne 0 ]; then
+        echo "Did not manage to  start transaction"
+        return 1
+    fi
+
+    mkdir -p /cvmfs/test.repo.org/new/c/d
+    echo "New file" > /cvmfs/test.repo.org/new/c/d/new_file.txt
+
+    echo "  Checking that we cannot open a transaction again"
+    cvmfs_server transaction test.repo.org
+    retcode=$?
+    if [ $retcode -ne 17 ]; then
+        echo "cvmfs_server transaction should tell us that we have a transaction already"
+        return 1
```

Comment:
We should use different codes for every return

Reply:
Done